docs/fundamentals/dashboard/overview.md

```diff
@@ -48,7 +48,7 @@ For more information, see the [Standalone .NET Aspire dashboard](standalone.md).
 
 ## Configuration
 
-The dashboard is configured when it starts up. Configuration includes frontend and OTLP addresses, the resource service endpoint, authentication, telemetry limits and more.
+The dashboard is configured when it starts up. Configuration includes frontend and OTLP addresses, the resource service endpoint, authentication, telemetry limits and more. The dashboard can also be [configured to allow browser telemetry](configuration.md#otlp-cors), which enables scenarios where telemetry data is sent directly to the dashboard from a browser.
```

Comment:
Configuring CORS is just one part of enabling browser telemetry. There is also configuring the OTLP endpoint, having the JavaScript OTEL SDK setup in the browser, configuring the right telemetry location (can be read from env vars), potentially sending the API key if auth is enabled (key can also be read from env vars).

Reply:
I've dug a lot deeper, and I have a much better understanding of the requirements. With that said, this PR now includes a lot more depth. I initially started with putting this into the configuration, but I'm somewhat leaning towards a new article under Dashboard, perhaps named Enable browser telemetry. Thoughts on that as an alternative to what I have thus far?